src/states/node.rs

```diff
@@ -615,6 +639,7 @@ impl Node {
                           self,
                           error,
                           pub_id);
+                    self.ban_client(&pub_id);
```

Comment:
Yes - I thought that would be OK for now since we're not catering for malicious nodes.